### PR TITLE
list_of_events: Document fs_dictmap_dict_write_uncertain event

### PR DESCRIPTION
**Pull request by cmouse**
JIRA: DOV-4119

### CODE REVIEW

File: source/admin_manual/list_of_events.rst [hunk 1]
```diff
@@ -2315,6 +2315,29 @@ indexes in metacache.
...
+.. versionadded:: 2.3.13
+
+The event is sent whenever a dict write is uncertain.
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| Inherits from fs_file                                                        |
++-----------------------+------------------------------------------------------+
++-----------------------+------------------------------------------------------+
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:2331]: Why two lines? Normally there's just one.

File: source/admin_manual/list_of_events.rst [hunk 2]
```diff
@@ -2315,6 +2315,29 @@ indexes in metacache.
...
+| path                  | Virtual FS path to the object (based on dict)        |
++-----------------------+------------------------------------------------------+
+| object_id             | Object ID in the storage                             |
++-----------------------+------------------------------------------------------+
+| cleanup               | success, failed or disabled                          |
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:2336]: Should have a longer explanation, like: If the uncertain write was attempted to be cleaned (deleted), result of the attempt.

File: source/admin_manual/list_of_events.rst [hunk 3]
```diff
@@ -2315,6 +2315,29 @@ indexes in metacache.
 fs-dictmap
 ----------
 
+
+fs_dictmap_dict_write_uncertain
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.13
+
+The event is sent whenever a dict write is uncertain.
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:2324]: Could explain a bit more, like: For example writes to Cassandra may eventually succeed even if the write initially appeared to fail.

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/admin_manual/list_of_events.rst (hunk 2 above)
Comment (sirainen) [reply source/admin_manual/list_of_events.rst:2336]: Maybe also point to the setting that can be used to enable/disable it?

File: source/admin_manual/list_of_events.rst [hunk 4]
```diff
@@ -2315,6 +2315,32 @@ indexes in metacache.
...
++-----------------------+-------------------------------------------------------+
+| Field                 | Description                                           |
++=======================+=======================================================+
+| Inherits from fs_file                                                         |
++-----------------------+-------------------------------------------------------+
+| path                  | Virtual FS path to the object (based on dict)         |
++-----------------------+-------------------------------------------------------+
+| object_id             | Object ID in the storage                              |
++-----------------------+-------------------------------------------------------+
+| cleanup               | success, failed or disabled. Indicates if uncertain   |
```
Comment (sirainen) [inline source/admin_manual/list_of_events.rst:2336]: Using \`\` quoting could make these choices clearer. Looks like they're missing from other places too .. but that could be fixed separately.

**Review by sirainen — CHANGES_REQUESTED** [review]

**Review by sirainen — APPROVED** [review]

**Review by thuovila — APPROVED** [review]

**Pull request merged**